rusk-abi/src/query.rs

```diff
@@ -4,64 +4,31 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
-use bytecheck::CheckBytes;
-use rkyv::{Archive, Deserialize, Serialize};
-
 mod public_input;
 pub use public_input::*;
 
-/// The type of the circuit to request a proof verification on.
-#[derive(Debug, Clone, Archive, Deserialize, Serialize)]
-#[archive_attr(derive(CheckBytes))]
-#[non_exhaustive]
-pub enum CircuitType {
-    /// Execute circuit with the given inputs and outputs
-    Execute(usize, usize),
-    /// Withdraw from contract transparent
-    WFCT,
-    /// Send to contract transparent
-    STCT,
-    /// Withdraw from contract obfuscated
-    WFCO,
-    /// Send to contract obfuscated
-    STCO,
-}
+pub(crate) enum Query {}
 
-/// Host query types offered by `rusk`.
-#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
-#[cfg_attr(not(feature = "host"), doc(hidden))]
-pub enum QueryType {
-    /// Perform a blake2b hash
-    Hash,
-    /// Perform a poseidon hash
-    PoseidonHash,
-    /// Verify a plonk proof
-    VerifyProof,
-    /// Verify a schnorr signature
-    VerifySchnorr,
-    /// Verify a BLS signature
-    VerifyBls,
+impl Query {
+    pub const HASH: &str = "h";
+    pub const POSEIDON_HASH: &str = "ph";
+    pub const VERIFY_PROOF: &str = "vp";
+    pub const VERIFY_SCHNORR: &str = "vs";
+    pub const VERIFY_BLS: &str = "vb";
```

Comment:
I am not sure about these shorter names, they are confusing, maybe we could leave them longer for the time being, until some final fine tuning is done

Reply:
They are indeed a little confusing. It's done to save on bytecode, which honestly isn't really necessary. Will make the change.